ingenialink/pdo.py

```diff
@@ -620,6 +620,7 @@ def _process_tpdo(self, input_data: bytes) -> None:
         for tpdo_map in self._tpdo_maps:
             map_bytes = input_data[: tpdo_map.data_length_bytes]
             tpdo_map.set_item_bytes(map_bytes)
+            input_data = input_data[tpdo_map.data_length_bytes :]
```

Comment:
keeping a running counter of extracted bytes would be more efficient than doing another slice for iteration, but this is ok